Cargo.lock

```diff
@@ -6462,7 +6462,7 @@ version = "2.27.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c2ef1dc036942fac2470fdb8a911f125404ee9129e9e807f3d12d8589001a38f"
 dependencies = [
- "log 0.4.14",
+ "log 0.3.9",
```

Comment:
Any idea why this one regressed?

Reply:
No, actually, will investigate

Reply:
It looks like this is instability being triggered by `protoc`'s curious decision to depend on [`log = "0.*"`](https://github.com/stepancheg/rust-protobuf/blob/a3dc27633a13a0ebfce34a054f1e7e8bb52fddac/protoc/Cargo.toml#L17). Should be fixed for us now, but we'll need to keep an eye on it.